db/migrations/public/20240918125758_create-return-cycles-view.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const viewName = 'return_cycles'
+
+exports.up = function (knex) {
+  return knex
+    .schema
+    .createView(viewName, (view) => {
+      view.as(knex('return_cycles').withSchema('returns').select([
+        'return_cycle_id AS id',
+        'start_date',
+        'end_date',
+        'due_date',
+        'is_summer',
+        'is_submitted_in_wrls',
```

Comment:
In order to match our conventions we would drop the `is_` prefix from these fields.

```suggestion
        'is_summer AS summer',
        'is_submitted_in_wrls AS submitted_in_wrls',
```